src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,102 @@
+import React,{useState, useEffect, FC} from 'react'
+import './Dashboard.scss'
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import Autocomplete from '@mui/material/Autocomplete';
+import Pagination from '@mui/material/Pagination';
+import Stack from '@mui/material/Stack';
+import ReactPaginate from 'react-paginate';
+import axios from 'axios';
+const options = ['Ho Chi Minh', 'Ha Noi', 'Da Nang', 'Can Tho'];
+const Dashboard : FC = () => {
+    const [jobs, setJobs] = useState([].slice(0, 100));
+    const [pageNumber, setPageNumber] = useState(0);
+    const jobsPerPage = 10; 
+    const pagesVisited = pageNumber * jobsPerPage;
+    const [searchTerm,setSearchTerm ] = useState("");
+        
+    useEffect(() => {
+        axios.get('https://6176370c03178d00173daae3.mockapi.io/api/api')
+        .then(res => {
+            setJobs(res.data)
+        }).catch(err => {
+            console.log(err);
+        })
+    },[])
+    
+    //Pagination
+    const pageCount = Math.ceil(jobs.length / jobsPerPage);
+    const changePage = ( {selected}:any ) => {
+            setPageNumber(selected);
+        };
+    
+    return (
+        <div className="dashBoard">
+            <h1>Có tất cả {jobs.length} IT Jobs For Chất Developers</h1>
+            <div className='search'>
+                <div className="search__form">
+                    <Box component="form" sx={{'& > :not(style)': { m: 1, width: '55ch' },

```

Comment:
sx={{'& > :not(style)': { m: 1, width: '55ch' }}

what is this used for?

Reply:
fixed